CFC_DataCollector/trip_generator/input.json

```diff
@@ -0,0 +1,15 @@
+{
+	"radius" : "1",
+	"starting centroids" :
+	[
+		"2703 Hallmark Dr Belmont Ca",
+		"ATT Park"
+
+	],
+	"ending centroids" :
+	[
+		"UC Berkeley",
+		"Morgan Hill"
+	],
+	"number of trips" : 100
```

Comment:
Suggestions for optional enhancements:

1. Allow people to specify the number of trips of each mode
1. Allow people to specify the total number of trips, and the probability distribution of each mode
1. Allow people to specify a set of tour models (bonus: will help us define what a tour model looks like)

@nephraim, can you think of other options that you might find useful?